cogs/coding.py

```diff
@@ -73,7 +77,13 @@ def get_h2_content(self, tag: NavigableString) -> str:
     }
 
     # TODO: lua, java, javascript, asm, c#
-    documented = {"c": _doc.c_doc, "cpp": _doc.cpp_doc, "haskell": _doc.haskell_doc, "python": _doc.python_doc}
+    documented = {
+        "c": _doc.c_doc,
+        "cpp": _doc.cpp_doc,
+        "haskell": _doc.haskell_doc,
+        "python": _doc.python_doc,
+        "rust": _doc.rust_doc,
+    }
 
     @commands.command(
         help="""
```

Comment:
130 and 131: snake_case
173: R1705
183: C0103: snake_case
189: rather use `if not text` (text can be "")
207: R1715: consider using `dict.get`
210: same
260: C0103